sdks/js/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@dust-tt/client",
+  "version": "1.0",
+  "author": "Dust",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "module": "dist/types.esm.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "build": "dts build",
+    "lint": "dts lint",
+    "format": "dts lint --fix",
+    "start": "dts watch"
+  },
+  "engines": {
+    "node": ">=20"
+  },
+  "devDependencies": {
+    "@tsconfig/recommended": "^1.0.3",
+    "@types/node": "^22.7.5",
+    "@types/uuid": "^9.0.7",
+    "tslib": "^2.6.2",
+    "typescript": "^5.4.5"
+  },
+  "dependencies": {
+    "@dust-tt/types": "file:../../types",
+    "dts-cli": "^2.0.5",
```

Comment:
should be in devDependencies

Reply:
WIll move (FYI i copy pasted from types, i'll update too)